clear-code/ieview-we
# Replace implementation of Chrome extension with ThinBridge extension

# Which issue(s) this PR fixes:

N/A

# What this PR does / why we need it:

This completely replaces the implementation of the Chrome extension with ThinBridge (MV3), because MV3 version must be loaded via GPO in enterprise environments and we have less motivation to maintain this addon with extra costs for non enterprise users.

And this keeps Firefox extension as MV2 - Mozilla announces that Firefox still support MV2 for more time.

# How to verify the fixed issue:

## The steps to verify:

1. Pack `chrome` contents as a CRX.
2. Configure your environment to install the built CRX via GPO. See documents of [ThinBridge](https://github.com/ThinBridge/ThinBridge/) or [BrowserSelector](https://gitlab.com/clear-code/browserselector) for more details.
3. Install ThinBridge from the installer.
4. Configure ThinBridge.
5. Under the ThinBridge insatlled directory, add the ID of the built CRX to the native manifest file for Chrome.
6. Browse the web with Chrome.

## Expected result:

Redirections defined by ThinBridge are proceeded as is.
